index.html

```diff
@@ -5,6 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Ornela Portfolio</title>
```

Comment:
Why don't we make this more descriptive: 🤔

- Better page titles:
     "Ornela's Portfolio| full-stack web developer" 👍